integreat_cms/cms/views/utils/stringify_list.py

```diff
@@ -0,0 +1,19 @@
+def stringify_list_with_different_delimiter(list_of_items):
+    """
+    This is a helper function that turns a list into a string using different delimiters.
+
+    :param list_of_items: The list that should be turned into a string
+    :type list_of_items: list
+
+    :return: The titles in a string using different delimiters
+    :rtype: str
+    """
+    titles_as_str = ""
+    for index, item in enumerate(list_of_items):
+        if int(index) == len(list_of_items) - 2:
+            titles_as_str += item + " & "
+        elif int(index) == len(list_of_items) - 1:
+            titles_as_str += item
+        else:
+            titles_as_str += item + ", "
+    return titles_as_str
```

Comment:
I assume there is some Python magic how to do it in one line using join, but I couldn't get it to work. Suggestion how to make this more concise are very welcome

Reply:
Ah yes, applied! I just renamed iter to _iter so we can get rid off the disable pylint and then pylint remarked that I can also remove the doc because for iter it's already obvious

Reply:
Ok, however, a underscore typically indicates that the variable is unused, right?
Maybe something like `iterable` makes more sense?

Reply:
And pylint did not mark the docstring as "obvious", but it complained the fact that you wanted to document an ignored parameter.
In the end, this would mean that this parameter is not visible in the docs, which is kind of sad when developers want to consult the documentation about it...

Reply:
> Ok, however, a underscore typically indicates that the variable is unused, right?

I didn't know this convention. [PEP8](
https://peps.python.org/pep-0008/#descriptive-naming-styles) recommends a trailing not a leading underscore in order not to interfere with any Python keywords. But I'm also fine with iterable, which to me is the more consistent solution

Reply:
Yeah, I guess the only "official" convention of the leading underscore is to indicate "private" methods and variables. However, for a function argument this does not make sense, since they only live in the local scope and are not accessible from the outside anyways.
So it just seems to be an unofficial convention to treat function arguments starting with an underscore as "dummy variables", e.g. pylint defines them here:

https://github.com/pylint-dev/pylint/blob/ae344fb04e9caddd69db4364d7c55b6e25395a0e/pylint/checkers/variables.py#L1213

I know a lot of projects that also use `_` as variable name for unused parameters, but this clashes with the gettext translation alias, so for Django projects this is not a good choice... (I noticed we still have a few occurrences of this pattern in our code base, maybe we should fix that some time :sweat_smile:)

The [Google style guide](https://google.github.io/styleguide/pyguide.html) even suggest completely to avoid underscores altogether and `del` the unused variables instead:

> Unused argument warnings can be suppressed by deleting the variables at the beginning of the function. Always include a comment explaining why you are deleting it. "Unused." is sufficient. For example:
> ```
> def viking_cafe_order(spam: str, beans: str, eggs: str | None = None) -> str:
>     del beans, eggs  # Unused by vikings.
>     return spam + spam + spam
> ```
> Other common forms of suppressing this warning include using `_` as the identifier for the unused argument or prefixing the argument name with `unused_`, or assigning them to `_`. These forms are allowed but no longer encouraged. These break callers that pass arguments by name and do not enforce that the arguments are actually unused.

As for the trailing underscore: Didn't know that, yeah also makes sense I guess, although I think e.g. `cls` is a lot more widespread than `class_`. In my opinion, variable names that have easy to understand alternatives should use them, but we could keep the trailing underscore option in mind when we face a similar problem in the future.

Reply:
That was very insightful, thank you! 

> (I noticed we still have a few occurrences of this pattern in our code base, maybe we should fix that some time 😅)

Yes, I will open an issue for that

>  I think e.g. cls is a lot more widespread than class_. In my opinion, variable names that have easy to understand alternatives should use them, but we could keep the trailing underscore option in mind when we face a similar problem in the future.

I agree!